.github/workflows/appsignals-python-e2e-test.yml

```diff
@@ -31,20 +31,31 @@ env:
   APP_SIGNALS_PYTHON_E2E_RE_SA_IMG: appsignals-python-django-remote-service
   METRIC_NAMESPACE: AppSignals
   LOG_GROUP: /aws/appsignals/eks
-#  ECR_OPERATOR_STAGING_REPO: ${{ vars.ECR_OPERATOR_STAGING_REPO }}
+  ECR_OPERATOR_STAGING_REPO: ${{ vars.ECR_OPERATOR_STAGING_REPO }}
```

Comment:
What's `vars`? Would we want to use secrets instead?

Reply:
This is referencing to `ECR_OPERATOR_STAGING_REPO` variable, which is used in ["Build And Upload Artifact" workflow](https://github.com/aws/amazon-cloudwatch-agent-operator/blob/8472f520bd50707494064564d33658b231518df0/.github/workflows/build-and-upload-release.yml#L8), the staging artifacts will be saved in this repo. So, I'm using the same variable here to make sure the image can be found.